navit/vehicle/demo/vehicle_demo.c

```diff
@@ -282,6 +296,7 @@ static struct vehicle_priv *vehicle_demo_new(struct vehicle_methods
     ret->cbl = cbl;
     ret->interval=1000;
     ret->config_speed=40;
+    ret->height=-10;
```

Comment:
is this a sane default? i would guess that about 2-3m is more appropriate

Reply:
This is the locations height above sealevel, not the vehicle height. The vehicles height is handled in the vehicle profile.

Reply:
Ah okay, than its valid, sure